app/src/pages/review.tsx

```diff
@@ -127,19 +129,19 @@ const Review: NextPage = () => {
       <ReviewCollection
         headerKey="Review.eligibilityTitle"
         editable={true}
-        editHref="/eligibility"
+        editHref={{ pathname: '/eligibility', query: reviewMode }}
```

Comment:
@karinamzalez I considered a number of options for how to better handle routing the user back to the review page and landed on the simplest one: setting a "mode" using a query string. A path like `/eligibility` will be interpreted as a user going through the form wizard in the normal flow. A path like `/eligibility?mode=review` will indicate that the user has gone through the whole form wizard and is now editing just that page.

I still need to add back in the guardrail checks, so that when the server sees `/eligibility?mode=review`, it will verify that the user is really allowed to be in review mode.